src/lex/replace.rs

```diff
@@ -439,3 +488,21 @@ fn stringify(args: Vec<Token>) -> Token {
         ret.trim()
     ))]))
 }
+
+fn concat(x: &Token, y: &Token, location: &Location) -> Option<Locatable<Token>> {
+    let mut lexer = Lexer::new(location.file, format!("{}{}", x, y), false);
```

Comment:
You said doing this means we can get rid of the LiteralParser trait, right? Do you want to do that here or in a follow-up PR?